src/plugins/files/server/saved_objects/file.ts

```diff
@@ -48,7 +48,7 @@ const properties: Properties = {
 export const fileObjectType: SavedObjectsType<FileMetadata> = {
   name: FILE_SO_TYPE,
   hidden: true,
-  namespaceType: 'multiple-isolated',
+  namespaceType: 'agnostic',
```

Comment:
So, we've been trying with @rudolf to find scenarios where this change could be dangerous, but I couldn't find any. So I think this may be fine. 

We may want to add a migration to get rid of / clear the `namespaces` attribute from the existing documents though?

Reply:
Based on [our telemetry](https://stack-telemetry.elastic.dev/s/kibana-core/app/dashboards#/view/c99350d0-4eb7-11ed-961a-43c1542c6e4b?_g=(filters:!(),query:(language:kuery,query:''),refreshInterval:(pause:!t,value:0),time:(from:'2022-10-20T12:37:14.482Z',to:now))) I can't see any clusters with `files` saved objects, so perhaps it's not generally available yet? 

Changing existing saved objects from `multiple-isolated` to `agnostic` is not something we've ever tried to do or that's tested. So if we expect that there could be deployments with existing `files` saved objects we should thoroughly test how it behaves.

Reply:
Right now there are no existing files integrations using saved objects in our code. In order to use files there must be a call to `registerFileKind`. Currently this is only in our `files_example` plugin.